rst.html.markdown

```diff
@@ -47,19 +47,27 @@ Title are underlined with equals signs too
 Subtitles with dashes
 ---------------------
 
-You can  put text in *italic* or in **bold**, you can "mark" text as code with double backquote ``print()``.
+You can put text in *italic* or in **bold**, you can "mark" text as code with double backquote ``print()``.
 
-Lists are as simple as in Markdown:
+Lists are similar to Markdown, but a little more involved.
+
+Remember to line up list symbols (like - or *) with the left edge of the previous text block, and remember to use blank lines to separate new lists from parent lists:    
 
 - First item
 - Second item
-    - Sub item
+
```

Comment:
@AndrewDiMola As you explained, blank lines are used to separate new lists from parent lists, then why would you put a blank line between the 'sub item' and its parent ie. 'second item'?

Reply:
@divayprakash nested lists always require a blank line between the parent and child element (e.g., the last parent list item and the first sublist item). More info: https://stackoverflow.com/questions/48523638/nested-sub-list-in-restructuredtext-with-code-blocks-not-working